runners/mlcube_singularity/mlcube_singularity/singularity_run.py

```diff
@@ -73,9 +73,12 @@ def merge(mlcube: DictConfig) -> None:
             except:
                 logger.warning("SingularityRun can't get singularity version (do you have singularity installed?).")
 
+            build_file = 'docker://' + d_cfg['image']
+            if 'tar_file' in d_cfg:
+                build_file = 'docker-archive://' + d_cfg['tar_file']
```

Comment:
The singularity user guide actually says that one should not use `//` for local archives, but apparently it still works. [Reference](https://sylabs.io/guides/latest/user-guide/singularity_and_docker.html#containers-in-docker-archive-files):

> We can convert this tar file into a singularity container using the docker-archive bootstrap agent. Because the agent accesses a file, rather than an object hosted by a service, it uses `:<filename>`, not `://<location>`.

Reply:
Oh, actually, this didn't work for me after all. I missed that I already had an singularity image in `workspace/.image` and the MLCube didn't have to build at all. So I would suggest to remove the `//` after all.
(Btw I'm using singularity v3.7.2)

Reply:
Hello, I just removed it, could you please test again?